invenio_app_ils/ui/src/invenio_app_ils/forms/components/AccordionField/AccordionField.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'formik';
+import { Accordion, Form, Icon, Segment } from 'semantic-ui-react';
+import './AccordionField.scss';
+
+export class AccordionField extends Component {
+  constructor(props) {
+    super(props);
+    this.fieldPath = props.fieldPath;
+    this.label = props.label;
+    this.required = props.required;
+    this.uiProps = props.uiProps;
+    this.state = { active: false };
+    this.iconActive = (
+      <Icon
+        name="angle down"
+        color="orange"
+        size="large"
+        className="form-accordion-icon"
+      ></Icon>
+    );
+    this.iconInActive = (
+      <Icon
+        name="angle right"
+        color="orange"
+        size="large"
+        className="form-accordion-icon"
```

Comment:
In our effort to clean the styles, I would just do `style={{float: 'right'}}` and kill the scss file, @kprzerwa wdyt?

Reply:
You are right I will remove this ;)